versions/v1.0.0/Social/Comment/main.adoc

```diff
@@ -39,6 +39,35 @@ interface Comment {
 }
 ----
 
+===== Fields
+
+`handle` *Optional*:: Handle/username of the user who is commenting, e.g. `looktanl.lens`.
+`body` *Required*:: Main content of the comment, e.g. `Cool.🌸`.
+`title` *Optional*:: Title of the comment.
+`summary` *Optional*:: Short summary of the comment.
+`media` *Optional*:: Array of media objects that are attached to the comment.
+`media.address` *Required*:: Url/address of the media object.
```

Comment:
Inconsistency in the capitalization of URLs (Url and URL)

Reply:
resolved.